hedera-node/hapi-fees/src/main/java/com/hedera/node/app/hapi/fees/usage/crypto/CryptoOpsUsage.java

```diff
@@ -192,13 +192,13 @@ public void cryptoCreateUsage(
 
         if (maxAutomaticTokenAssociations > 0) {
             baseSize += INT_SIZE;
+            accumulator.addRbs(maxAutomaticTokenAssociations * lifeTime * CREATE_SLOT_MULTIPLIER);
```

Comment:
In my understanding, **all** automatic token association slots should be free.

Reply:
Do we want to completely remove this as I don't see a way to check the feature flag there?